controllers/datadogagent/feature/orchestratorexplorer/feature.go

```diff
@@ -66,7 +66,7 @@ type orchestratorExplorerFeature struct {
 	processAgentRequired bool
 }
 
-const NoProcessAgentMinVersion = "7.51.0"
+const NoProcessAgentMinVersion = "7.51.0-0"
```

Comment:
Had to be done in order for this check to work properly with prereleases. Otherwise, `7.53.0-rc.1` >= `7.51.0` would evaluate as false. Also done this way [here](https://github.com/DataDog/datadog-operator/blob/7db14b9f28692d91a33d8a483169e11119c71171/controllers/datadogagent/feature/kubernetesstatecore/feature.go#L71).